lib/reindex/s3_bucketd.py

```diff
@@ -314,6 +322,11 @@ def update_redis(client, resource, name, obj_count, total_size):
     client.set(obj_count_key + ':counter', obj_count)
     client.set(total_size_key + ':counter', total_size)
 
+@as_set
```

Comment:
Not really sure making a generator method return a set this way is really ideal.
You can already turn generators like this into sets by just using `set(gen_func(...))` and it removes the option of using this as a normal generator if needed.